Towny/src/main/java/com/palmergames/bukkit/towny/tasks/PlotClaim.java

```diff
@@ -206,30 +211,32 @@ private boolean residentClaim(WorldCoord worldCoord) throws TownyException {
 		if (owner != null && price > 0)
 			TownyMessaging.sendPrefixedTownMessage(town, Translatable.of("msg_buy_resident_plot", resident.getName(), owner.getName(), townBlock.getPlotPrice()));
 
-		claimTownBlockForResident(townBlock);
-		return true;
+		return claimTownBlockForResident(townBlock);
 	}
 
-	private void claimTownBlockForResident(TownBlock townBlock) {
-		townBlock.setResident(resident);
+	private boolean claimTownBlockForResident(TownBlock townBlock) {
+		if(!townBlock.setResident(resident)) return false;

```

Comment:
We don't usually put our returns on the same line, I'd like all of the instances of it in this PR changed to being two separate lines.